Slim/Http/Response.php

```diff
@@ -499,7 +499,7 @@ public function count()
      *
      * @return \Slim\Http\Headers
```

Comment:
phpstan complains about this, I think phpdoc's @return have to match the actual data type.